software/ntfy/egg-ntfy.json

```diff
@@ -15,14 +15,14 @@
     "file_denylist": [],
     "startup": ".\/ntfy serve --config server\/server.yml",
     "config": {
-        "files": "{\r\n    \"server\/server.yml\": {\r\n        \"parser\": \"yaml\",\r\n        \"find\": {\r\n            \"listen-http\": \":{{server.build.default.port}}\",\r\n            \"cache-file\": \".\/cache.db\"\r\n        }\r\n    }\r\n}",
```

Comment:
Wy did you remove this?

Reply:
I removed this because this would only show when the server was started for the first time which I don't think is the best so I moved the server.yml creation into the init script.

Reply:
The files run on every startup and should be there as you must force the port it uses, and hard setting the cache db in it is also a good idea. The config just has to be in that place before the server starts for the first time.

Reply:
Even though I create the file in the init script should I still keep it in the files? I just assumed it would be better as when the user installs the server they don't have to start it to see the configuration file

Reply:
End users do not see this configuration logic. it is expected that all ports an application uses are locked and users can not change it so they copy a config from somewhere, and it has another port set that it reverts that line back. else host will get flooded with support tickets.